src/components/AddressSearch.js

```diff
@@ -132,8 +132,8 @@ const AddressSearch = (props) => {
             street: props.value ? props.value.trim() : '',
             city: city || cityFallback,
             zipCode,
-            state,
             country: '',
+            state,
```

Comment:
What are we doing here to ensure the order doesn't get updated in the future? Will a comment be helpful?